--- a/maven-plugin-tools-generators/src/main/java/org/apache/maven/tools/plugin/generator/PluginHelpGenerator.java
+++ b/maven-plugin-tools-generators/src/main/java/org/apache/maven/tools/plugin/generator/PluginHelpGenerator.java
@@ -22,6 +22,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.io.FileOutputStream;
 import java.io.OutputStreamWriter;
 import java.io.StringWriter;
 import java.io.Writer;
@@ -31,7 +32,6 @@
 import org.codehaus.plexus.logging.AbstractLogEnabled;
 import org.codehaus.plexus.logging.Logger;
 import org.codehaus.plexus.logging.console.ConsoleLogger;
-import org.codehaus.plexus.util.io.CachingOutputStream;
 import org.codehaus.plexus.velocity.VelocityComponent;
 
 import static java.nio.charset.StandardCharsets.UTF_8;
@@ -82,7 +82,7 @@
 
             String helpClassSources = getHelpClassSources(getPluginHelpPath(mavenProject));
 
-            try (Writer w = new OutputStreamWriter(new CachingOutputStream(helpClass), UTF_8)) {
+            try (Writer w = new OutputStreamWriter(new FileOutputStream(helpClass), UTF_8)) {
                 w.write(helpClassSources);
             }
         } catch (IOException e) {

Diff finished.  Sun Feb 18 13:31:02 2024
