--- a/src/cider/nrepl/middleware/slurp.clj
+++ b/src/cider/nrepl/middleware/slurp.clj
@@ -19,14 +19,12 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 (def known-content-types
-  (->> (io/resource "cider/nrepl/content-types.edn")
-       (io/reader)
-       (java.io.PushbackReader.)
-       (edn/read)
-       (mapcat (fn [[content-type exts]]
-                 (for [ext exts]
-                   [ext content-type])))
-       (into {})))
+  {"edn" "text/edn",
+   "clj" "text/clojure",
+   "cljc" "text/clojure",
+   "cljs" "text/clojurescript",
+   "yml" "text/yaml",
+   "yaml" "text/yaml"})
 
 (defn- split-last
   [^String to-split ^String where]

Diff finished.  Fri Dec  2 00:10:05 2022
